camunda-formio-tasklist-vue/src/components/layout/Header.vue

```diff
@@ -37,6 +38,7 @@ import BaseMixin from "../mixins/BaseMixin.vue";
 import { FilterPayload } from "../../models/FilterPayload";
 import FormListModal from "../form/FormListModal.vue";
 import { Payload } from "../../models/Payload";
+import { TaskListSortType } from "@/services";
```

Comment:
Shouldn't we follow the style guide recommendation of importing relative to filenames?

https://github.com/AOT-Technologies/forms-flow-ai-extensions/blob/master/camunda-formio-tasklist-vue/docs/StyleGuide.md